controllers/v1/userExtensionController.js

```diff
@@ -0,0 +1,247 @@
+/**
+ * name : userExtension.js
+ * author : Rakesh Kumar
+ * created-date : 19-march-2020
+ * Description  User related information.
+ */
+
+
+/**
+ * dependencies
+ * 
+ */
+const userExtensionHelper = require(ROOT_PATH + "/module/userExtension/helper");
+
+
+
+/**
+    * UserExtension
+    * @class
+*/
+
+
+
+module.exports = class UserExtension extends Abstract {
+
+    constructor() {
+        super(userExtensionSchema);
+   
+    }
+
+    /**
+     * @apiDefine errorBody
+     * @apiError {String} status 4XX,5XX
+     * @apiError {String} message Error
+     */
+
+    /**
+     * @apiDefine successBody
+     *  @apiSuccess {String} status 200
+     * @apiSuccess {String} result Data
+     */
+
+
+    static get name() {
+        return "userExtension";
+    }
+
+
+
+      /**
+  * @api {get} /user-management-service/api/v1/userExtension/getProfile/{{userId}} Get user profile
```

Comment:
@rakeshSgr - The URL is /user-management/api/v1

Reply:
resolved